### DISCUSSION TITLE
資の井戸端

### DISCUSSION THREAD

**Discussion by NiiiNonno** [NiiiNonno/Assets#16] [General]
号にならない些細な質問、思い付き、その他何でも。

積要、必要になり次第分割し、機能を明確にする次。

**Comment by NiiiHaruto** [comment]
Scrollの環境依存性が強くて、エンディアン処理とか、構造体の配置とかの問題どうするの。

**Reply by NiiiNonno** [reply]
74a03682a1ad4d54d197c38c68614f3423746b4f にて補助クラスを置いたから、これを中継に使う巻子を作ればエンドユーザの任意で構造体の配置を標準化できる。
